src/main/kotlin/no/nav/familie/tilbake/api/ForvaltningController.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.tilbake.api
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.tilbake.forvaltning.ForvaltningService
+import no.nav.familie.tilbake.sikkerhet.Behandlerrolle
+import no.nav.familie.tilbake.sikkerhet.Rolletilgangssjekk
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.math.BigInteger
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/forvaltning/")
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class ForvaltningController(private val forvaltningService: ForvaltningService) {
+
+    @PutMapping(path = ["/behandling/{behandlingId}/kravgrunnlag/{kravgrunnlagId}/v1"],
+                produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Rolletilgangssjekk(minimumBehandlerrolle = Behandlerrolle.VEILEDER,
+                        handling = "Henter korrigert kravgrunnlag fra økonomi og oppdaterer kravgrunnlag431",
+                        henteParam = "behandlingId")
+    fun korrigerKravgrunnlag(@PathVariable("behandlingId") behandlingId: UUID,
+                             @PathVariable("kravgrunnlagId") kravgrunnlagId: BigInteger): Ressurs<String> {
```

Comment:
De't som står inne i parentesene til @PathVariable er for å overstyre parameternavnet. Siden du bare gjentar parameternavnet, så er innholdet unødvendig.